CA_DataUploaderLib/ServerUploader.cs

```diff
@@ -558,6 +559,38 @@ public static async Task<PlotConnection> Establish(string loopName, byte[] publi
                 }
             }
 
+            private static async Task<(int plotId, string plotName)> GetPlotIDAsyncWithRetries(string loopName, HttpClient client, string loginToken, byte[] publicKey, byte[] signedVectorDescription, CancellationToken cancellationToken)
+            {
+                int failureCount = 0;

```

Comment:
where is failureCount  incremented?

Reply:
No where 🙃. Will fix.

Reply:
It's also an issue in GetLoginTokenWithRetries